app/scripts/utils/url.ts

```diff
@@ -0,0 +1,27 @@
+import React, { MouseEventHandler } from 'react';
+import { getBoolean } from 'veda';
+import { LinkProps } from 'react-router-dom';
+
+
+export const getLinkProps = (
+    linkTo: string,
+    as?: React.ForwardRefExoticComponent<LinkProps & React.RefAttributes<HTMLAnchorElement>>,
+    onClick?: (() => void) | MouseEventHandler
+  ) => {
+    const externalLinksInNewTab = getBoolean('externalLinksInNewTab');
+    const isExternalLink = /^https?:\/\//.test(linkTo);
+    return isExternalLink
+    ? {
+        href: linkTo,
+        to: linkTo,
```

Comment:
Will this work? I tested in the deploy preview and looks like the external link I clicked wasn't opening in a new tab?

![Screenshot 2024-03-28 at 5 20 34 PM](https://github.com/NASA-IMPACT/veda-ui/assets/30272083/06585689-b3e2-42c8-afb8-1fff96735b75)

Reply:
@sandrahoang686 that's because we also need externalLinksInNewTab set to true on veda-config/veda.config.js. 
It is working on veda-config-ghg (https://github.com/US-GHG-Center/veda-config-ghg/pull/326) and also on my local machine.
I had to set it this way to prevent typescript error which was expecting "to" prop from the function